src/mozillavpn.cpp

```diff
@@ -1282,6 +1282,15 @@ void MozillaVPN::registerUrlOpenerLabels() {
             : settingsHolder->captivePortalIpv4Addresses().first());
   });
 
+  uo->registerUrlLabel("downloadExtension", []() -> QString {
+    // TODO: This link MUST be updated with a link to our extension
+    return "https://addons.mozilla.org/en-US/firefox/";
+  });
+
+  uo->registerUrlLabel("downloadFirefoxWindows", []() -> QString {
+    return "https://www.mozilla.org/firefox/windows/";
```

Comment:
If we do `https://www.mozilla.org/firefox/new/` it will take them directly to the desktop install page and bedrock will automagically determine the platform. We probably want to attach some UTM parameters to this link too.

Reply:
Thanks, updated the URL (and name). I left a message for Santiago on the associated ticket to see if he wants UTM params on either of these links.